### DISCUSSION TITLE
Revisar Solicitud 2636 o 2635

### DISCUSSION THREAD

**Discussion by ignacioCotalker** [Cotalker/documentation#22] [2. Questions and Answers] [answered]
Holaa, tengo estas dos solicitudes (son iguales). No se si me pueden dar info de cuando me puedan ayudar a ver si se puede corregir, gracias.

**Comment by acedward** [comment, marked as answer]
@ignacioCotalker por favor realizar este tipo de solicitud por canales internos. Gracias : ) 

Si requieres esto para un cliente por favor levantar el requerimiento con @jdurandpaz y @guillevalin para que acuerden un deadline / SLA

**Reply by jdurandpaz** [reply]
cual es el canal interno?

**Reply by ignacioCotalker** [reply]
@acedward hice la pregunta aca pq el canal de ing se encuentra bloqueado

**Reply by acedward** [reply]
@jdurandpaz no hay un canal para esto, pueden conversarlo - porque deberian ser casos excepcionales/importantes que están bloqueando un cliente o implementación. Ojalá dejen un email enviado con los compromisos de los equipos.

**Reply by guillevalin** [reply]
@jdurandpaz @ignacioCotalker los tickets siempre me los pueden preguntar a mi personalmente o por el canal de proyectos.

**Reply by acedward** [reply]
@jdurandpaz @ignacioCotalker @SeIturra idealmente dejen por escrito los deadlines comprometidos para issues que son roadblocks para clientes y/o usuarios
